pkg/rancher-desktop/config/settings.ts

```diff
@@ -104,6 +101,19 @@ export const defaultSettings = {
 
 export type Settings = typeof defaultSettings;
 
+/**
+ * Lockable default settings used for validating deployment profiles.
+ * Data values are ignored, but types are used for validation.
+ */
+export const lockableDefaultSettings = {
```

Comment:
This should be moved to `backgroundProcess.ts` and then it doesn't need to be exported

Reply:
It makes more sense to me that the two settings structures that are closely related and will change with different versions would be together in the same settings file.

Reply:
True, but I think `lockedDefaultSettings` is only going to be used by the profile reader, even though its structure is a lot like `defaultSettings`.